### ISSUE TITLE
Github Actions automatically run and fail when sync fork

### ISSUE THREAD

**Issue by Imss27** [huggingface/transformers#34329]
### System Info

N/A

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

Sync fork in my repo will cause such problem and I keep getting emails about failed CI jobs. 

### Expected behavior

Github actions CI should not run automatically for my forked repo.

**Comment by LysandreJik** [comment]
I think that's more an issue with settings in your git clone no?

**Comment by LysandreJik** [comment]
If you're aware of any settings we could change to make your life simpler, let us know!

**Comment by Imss27** [comment]
That's strange. Actually I just forked the repo and didn't commit anything.🤔

Anyway, just wonder if anyone has similar issues, I am not a CI expert, sorry can't be more helpful.

I attached a picture for your reference, looks like it regards some commits as my commits.
![ci](https://github.com/user-attachments/assets/a00fddd5-3308-49c5-b312-724b9ed40e56)

Feel free to close this issue if we can't find the reason for now.
